packages/react/src/components/builder-content.component.tsx

```diff
@@ -199,6 +199,11 @@ export class BuilderContent<ContentType extends object = any> extends React.Comp
               data: match,
               loading: false,
             });
+            const isPreviewing = (builder.editingModel || builder.previewingModel) === this.name;
+            if (!this.props.inline && this.firstLoad && isPreviewing) {
```

Comment:
Can you add a comment here to clarify the intention behind this change? To me it is not clear just from the code